code/datums/gamemodes/game_mode.dm

```diff
@@ -719,7 +719,9 @@ Sensors indicate [numXenosShip ? "[numXenosShip]" : "no"] unknown lifeform signa
 
 	for(var/i in GLOB.human_mob_list)
 		var/mob/living/carbon/human/H = i
-		var/datum/job/job = SSjob.GetJob(H.job)
+		var/datum/job/job = SSjob.GetJob(H.mind ? H.mind.assigned_role : H.job)
```

Comment:
We are moving away from `mind.assigned_role`, so try to avoid it.
Every job with a larvaworth should have a job referenced in the mob already.

Reply:
Got it, i was wondering why the job might be blank on some mobs, and instead of just skipping them trying extra hard to find one (by looking at `assigned_role`).

Will revert this.